build-system/tasks/runtime-test/helpers.js

```diff
@@ -39,17 +55,13 @@ const LARGE_REFACTOR_THRESHOLD = 50;
  */
 function extractCssJsFileMap() {
   //TODO(estherkim): consolidate arg validation logic
-  if (!fs.existsSync(extensionsCssMapPath)) {
-    log(
-      red('ERROR:'),
-      'Could not find the file',
-      cyan(extensionsCssMapPath) + '.'
-    );
+  if (!fs.existsSync(EXTENSIONSCSSMAP)) {
+    log(red('ERROR:'), 'Could not find the file', cyan(EXTENSIONSCSSMAP) + '.');
     log('Make sure', cyan('gulp css'), 'was run prior to this.');
     process.exit();
   }
```

Comment:
I wonder if this block is redundant now that we run `css()` inside `maybeBuild()`. No harm in leaving it in though.